python/lambda.py

```diff
@@ -1,16 +1,129 @@
-from api.index_handler import IndexHandler
+import json
+import os
+
+import mysql.connector
+from mysql.connector import errorcode
+
 from api.help import help_message
-from api.message_handler import MessageHandler
-from lib.datastore import DataStore
+from api.index_handler import IndexHandler
 from api.inventory_handler import InventoryHandler
+from api.message_handler import MessageHandler
 from lib.app_config import AppConfig
-from lib.validate.configuration import Configuration
+from lib.app_logging import AppLogging
+from lib.behaviors.repository import Repository
 from lib.http_utils import HttpUtils
 from lib.tron_response import TronResponse
-from lib.app_logging import AppLogging
-from lib.behaviors import repository
+from lib.validate.configuration import Configuration
+from repository.database_connection_info import DatabaseConnectionInfo
+from repository.database_inventory_repository import \
+    DatabaseInventoryRepository
+from repository.file_inventory_repository import FileInventoryRepository
+from repository.i_database_connector import IDatabaseConnector
+from repository.mysql_connector import MySqlConnector
+
+
+def setup_database(database_connector: IDatabaseConnector, database_connection_info: DatabaseConnectionInfo, inventory_data):
```

Comment:
Same as my other comment. This feels out of place in the entry point file.

Reply:
i can try to clean this up. should be possible to move this to a better home. 

i was trying to avoid having duplicate files, since the lambda cant use the DI container but I need roughly the same code anyways (same functionality, just without importing DI, which the other setup_database has and why i cant use it) .

Reply:
Not that I want to change things, but is the Container package adding enough value to justify having to repeat the database connection logic?

Reply:
i dont think they are / have to be exclusive, this is just the quickest thing i had in mind in terms of implementation. i very much would like to keep & use the DI container, so ill try improve this.